src/test/java/seedu/progresschecker/logic/parser/GoToTaskUrlCommandParserTest.java

```diff
@@ -0,0 +1,43 @@
+package seedu.progresschecker.logic.parser;
+
+import static seedu.progresschecker.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.progresschecker.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.progresschecker.logic.parser.ParserUtil.MESSAGE_INVALID_INDEX_OR_FORMAT;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INDEX_FIRST_TASK;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INDEX_FIRST_TASK_INT;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INVALID_CHARSET;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INVALID_NEGATIVE;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INVALID_ZERO;
+
+import org.junit.Test;
+
+import seedu.progresschecker.logic.commands.GoToTaskUrlCommand;
+
+//@@author EdwardKSG
+public class GoToTaskUrlCommandParserTest {
+
+    private GoToTaskUrlCommandParser parser = new GoToTaskUrlCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsGoToTaskUrlCommand() {
+        assertParseSuccess(parser, INDEX_FIRST_TASK, new GoToTaskUrlCommand(INDEX_FIRST_TASK_INT));
+    }
+
+    @Test
+    public void parse_invalidArgs1_throwsParseException() {
```

Comment:
Maybe you might want to specify what kind of invalid args you are testing for in these tests instead of numbering them? e.g. ```parse_invalidNegativeIndex_throwParseException```, ```parse_invalidZeroIndex_throwParseException```, ```parse_invalidCharset_throwParseException``` (I'm not sure if it's testing for invalid index so do amend my given example accordingly with what you see fit!)

Reply:
yep haha later I realized this and changed name for the other tests, but lazy to do this. I'll change this now :)